libraries/standard/serializer/test/unit/iot_tests_serializer_cbor.c

```diff
@@ -431,6 +431,14 @@ TEST( Serializer_Unit_CBOR, Encoder_map_nest_array )
     TEST_ASSERT_TRUE( cbor_value_at_end( &arrayElement ) );
 }
 
+/* JSON Representation:
```

Comment:
(Nit): I have tried to use the "_CBOR diagnostic notation/representation_" words (as mentioned on cbor.me) instead of JSON to avoid any possible confusion.

Reply:
Done.